packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec

```diff
@@ -5,16 +5,16 @@
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 2.1.1
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: Hooks Roadie into your Rails application to help with email generation
 Group: Development/Languages
 License: MIT
 URL: https://github.com/Mange/roadie-rails
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(railties) >= 5.1
-Requires: %{?scl_prefix}rubygem(railties) < 5.3
+Requires: %{?scl_prefix}rubygem(railties) >= 6.0
+Requires: %{?scl_prefix}rubygem(railties) < 7.0
```

Comment:
This doesn't seem to match the gemspec - https://github.com/Mange/roadie-rails/blob/master/roadie-rails.gemspec#L25